scripts/compile-win.bat

```diff
@@ -1 +1 @@
-"C:\MinGW\bin\c++.exe" "src\bin\WinKeyServer\main.cpp" -o "bin\WinKeyServer.exe" -static-libgcc -static-libstdc++
```

Comment:
Thinking this may be installation specific, which would be a pain in the butt... Feel it's best if I test whether I can compile it first before merging

Reply:
I installed mingw via chocolatey (`choco install mingw`) and `c++` was not in `C:\MinGW\bin\` but it was on my `PATH`, so I had to change this. Also with `-static-libgcc -static-libstdc+` I could not start newly compiled `WinKeyServer.exe` because it tries to link to nonexistent `libwinpthread-1.dll`. Changing `-static-libgcc -static-libstdc+` to `-static` helped this.